lib/engine/game/g_18_usa/game.rb

```diff
@@ -753,10 +753,8 @@ def check_connected(route, token)
             end
           end
 
-          paths_ = route.paths.uniq
-          token = blocked if blocked
-
-          return if token.select(paths_, corporation: route.corporation).size == paths_.size
+          # no need to check whether tokened out because of the above
+          return if route.ordered_paths.each_cons(2).all? { |pair| pair[0].connects_to?(pair[1], nil) }
```

Comment:
couldn't this be cleaned up? just be like route.connected? with an arg to pass in corp or nil,

Reply:
Yes, I meant to do this, but got side-tracked.